src/OpenTelemetry.Resources.ProcessRuntime/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* Removed `process.runtime.description` which has never been a support semconv attribute
```

Comment:
Please check this section: https://github.com/open-telemetry/semantic-conventions/blob/v1.36.0/docs/resource/process.md#process-runtimes